tools/zoslog/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"sync"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/containerd/containerd/runtime/v2/logging"
+	"github.com/threefoldtech/zos/pkg/container/logger"
+)
+
+var (
+	// RootDir is the contd module root directory
+	RootDir = "/var/cache/modules/contd/"
+
+	// LogsDir points to logs directory
+	LogsDir = "logs"
+
+	// ConfigDir points to config (logs settings) directory
+	ConfigDir = "config"
+)
+
+func main() {
+	logging.Run(runlog)
+}
+
+func runlog(ctx context.Context, config *logging.Config, ready func() error) error {
+	// initializing container logger
+	cfgfile := path.Join(RootDir, ConfigDir, config.Namespace, fmt.Sprintf("%s-logs.json", config.ID))
+
+	// load config saved by contd
+	logs, err := logger.Deserialize(cfgfile)
+	if err != nil {
+		return err
+	}
+
+	// initializing logs endpoints
+	loggers := logger.NewLoggers()
+
+	// create local default logs directory
+	local := path.Join(RootDir, LogsDir, config.Namespace)
+	if err = os.MkdirAll(local, 0755); err != nil {
+		return err
+	}
+
+	// hardcode local logfile
+	filepath := path.Join(local, fmt.Sprintf("%s.log", config.ID))
+	fileout, fileerr, err := logger.NewFile(filepath, filepath)
```

Comment:
do we still want to logs to disk ? I'm not sure

Reply:
It's not enabled by default now (see line 67)